VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -148,4 +122,110 @@ public void onClickAddComment(@NonNull DiscussionComment comment) {
     public void onClickViewComments(@NonNull DiscussionComment comment) {
         router.showCourseDiscussionComments(getActivity(), comment, discussionThread.isClosed());
     }
+
+    public static class DiscussionResponsesLoader implements
+            InfiniteScrollUtils.PageLoader<DiscussionComment> {
+        private InfiniteScrollUtils.PageLoadCallback<DiscussionComment> callback;
+
+        private Context context;
```

Comment:
Note that local variables can be annotated as `NonNull` and `Nullable` as well. So `context` and `threadId` can be defined as `NonNull`, while the tasks and `unendorsedComments` can be declared as `Nullable`.